src/browser-lib/single-spa-angular.ts

```diff
@@ -189,3 +210,5 @@ type SingleSpaAngularOpts = {
   domElementGetter?(): HTMLElement;
   AnimationEngine?: any;
 }
+
+export { getSingleSpaExtraProviders } from './extra-providers';
```

Comment:
I thought that it wasn't possible to export providers like this without switching to ng-packagr?

Reply:
Update: discussed this with Artur on slack and we sorted this out. I was mistaken.